src/pybind/mgr/dashboard_v2/tests/test_auth.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import
+
+import time
+from cherrypy.lib.sessions import RamSession
+from cherrypy.test import helper
+from mock import patch
+
+from ..auth import Auth
+from ..module import Module, cherrypy
+
+class AuthTest(helper.CPWebCase):
+    @staticmethod
+    def setup_server():
+        module = Module('dashboard', None, None)
+        auth = Auth(module)
+        cherrypy.tools.autenticate = cherrypy.Tool('before_handler', auth.check_auth)
+        cherrypy.tree.mount(auth, "/api/auth",
+                            config={'/ping': {'tools.autenticate.on': True}})
```

Comment:
I would remove the `ping` method from the Auth class, and would create a new endpoint class here with the ping method just to test it.

Reply:
Done